tests/Integration/Behaviour/Features/Context/Domain/AbstractDomainFeatureContext.php

```diff
@@ -146,4 +150,23 @@ protected function parseLocalizedArray(string $parsedArray): array
 
         return $localizedArray;
     }
+
+    /**
+     * Used in scenarios which have horizontal properties table
+     *
+     * @param TableNode $table
+     *
+     * @return array
+     *
+     * @throws RuntimeException
+     */
+    protected function extractFirstHorizontalRowFromProperties(TableNode $table): array
```

Comment:
in which case a row is not horizontal? :thinking:

Reply:
https://github.com/PrestaShop/PrestaShop/blob/0facef127b6d512cce953c5e2c693705af46fdec/tests/Integration/Behaviour/Features/Scenario/Customer/customer_management.feature#L8-L12

Reply:
Actually all our behat tests are written this way and they use `$data = $node->getRowsHash();` to get the data, so I'm not sure it add interest to change this for those tests, especially since you only get the data for one row
This only multiply the way we build the test, so for somone who would want to add some it adds complexity without additional value

Reply:
Yes, that's true Behat gives the ability to return table hash, formed by rows or columns. So this kind of method and it's naming I've created has no additional value. Thank you for noticing :)